app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -1,30 +1,15 @@
-import React, { useState } from "react";
+import React from "react";
 
 import { Alert } from "neetoui";
 
-const DeleteAlert = ({ onClose }) => {
-  const [deleting, setDeleting] = useState(false);
-
-  const handleDelete = async () => {
-    try {
-      setDeleting(true);
-      onClose();
-    } catch (error) {
-      logger.error(error);
-      setDeleting(false);
-    }
-  };
-
-  return (
-    <Alert
-      isOpen
-      isSubmitting={deleting}
-      message="Are you sure you want to continue? This cannot be undone."
-      title="Delete note?"
-      onClose={onClose}
-      onSubmit={handleDelete}
-    />
-  );
-};
+const DeleteAlert = ({ onClose, handleDelete }) => (
+  <Alert
+    isOpen
+    message="Are you sure you want to continue? This cannot be undone."
+    title="Delete note?"
+    onClose={onClose}
+    onSubmit={handleDelete}
+  />
+);
```

Comment:
Go through the reviews I had pointed out via Github in one of your other PRs and make the change.

Reply:
Changed in previous PR

Reply:
The message you had added was too verbose. Keep it compact yet on point:

```
Are you sure you want to delete the note "${note.name}"? This action cannot be undone."
```

Reply:
ok, sure this one is on point and compact as well.